core/extraction/towns_fund_round_four.py

```diff
@@ -48,6 +50,9 @@ def ingest_round_four_data_towns_fund(df_ingest: dict[str, pd.DataFrame]) -> dic
     towns_fund_extracted["Project Progress"] = r3.extract_project_progress(
         df_ingest["3 - Programme Progress"], project_lookup, round_four=True
     )
+    towns_fund_extracted["Programme Management"] = extract_programme_management(
+        df_ingest["4a - Funding Profiles"], programme_id
+    )
```

Comment:
Following our current pattern this should be defined as a round 3 extraction function to be reused here in round 4, as we want to extract this for both R3 and R4.